src/photos/ducks/clustering/files.js

```diff
@@ -41,41 +35,33 @@ export const getFilesFromDate = async (
   return files
 }
 
-export const getAllPhotos = async () => {
-  const files = await cozyClient.data.findAll(DOCTYPE_FILES)
+export const getAllPhotos = async client => {
+  // This does not use pagination but is significantly faster
+  // Note this is only used once, for init
+  const results = await client
+    .getStackClient()
+    .collection('io.cozy.files')
+    .all({ limit: null })
+  const files = results.data
```

Comment:
Why are we using stackClient? By doing this, we shortcut the PouchLink (so no offline for instance) :/. Is it only for performance reason?

Reply:
Is it possible to call the [all](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/DocumentCollection.js#L40) method without stackClient? 
I wanted to use this to request the stack on the `_all_docs` endpoint, which seemed dramatically faster for large datasets.

I didn't know about the bypassing of PouchLink, however this code  it is only used for clustering init, where we query all the files to compute the parameters, so it's probably not crucial for offline?

Reply:
I think there is no other way to call `_all_docs` :(

Reply:
Have you tried with .limit(null) on a query passed to cozyClient.query() ?

Reply:
Right, it works with a `limitBy(null)`